package.json

```diff
@@ -65,5 +65,6 @@
   "volta": {
     "node": "16.18.0",
     "yarn": "3.2.4"
-  }
+  },
+  "packageManager": "yarn@2.4.3"
```

Comment:
Yarn version is defined by volta, when we redefine it somewhere else it creates conflicts and breaks yarn